src/main/cpp/include/loader/tiledb_loader_file_base.h

```diff
@@ -114,8 +114,12 @@ class BufferReaderBase : public virtual GenomicsDBImportReaderBase
     {
       if(src == 0)
         return 0u;
-      if(m_num_valid_bytes_in_buffer+num_bytes > m_buffer.size())
+      if(m_num_valid_bytes_in_buffer+num_bytes > m_buffer.size()) {
+        std::cout << "Buffer resized from " << m_buffer.size()
```

Comment:
Change to std::cerr - many tools write real output to std::cout - informational messages must be sent to std::cerr